graphene_django_extras/mutation.py

```diff
@@ -236,9 +236,17 @@ def delete(cls, root, info, **kwargs):
         if old_obj := get_Object_or_None(
             cls._meta.model, info, type='delete', pk=pk
         ):
-            old_obj.delete()
-            old_obj.id = pk
-            return cls.perform_mutate(old_obj, info)
+            try:
+                old_obj.delete()
+                old_obj.id = pk
+                return cls.perform_mutate(old_obj, info)
+            except RuntimeError as e:
```

Comment:
Could you make this a generic exception...? Also won't the same be needed in the update as well?

Reply:
is anywhere we are raising generic exceptions? if yes plz let me know the case why we have to catch the generic exceptions.  need to check in for an update.

Reply:
In many places, while CRUD ops we have raised ValueErrors and Validation errors as well. So was concerned if these errors will get caught here or not.

Reply:
Also, won't this change need to be done on update and create methods as well?